backend/src/main/java/com/example/backend/service/quiz/QuizService.java

```diff
@@ -140,6 +140,16 @@ else if (level < 7) {
             }
         }
         else {
+            Integer languageId = quiz.getLanguageId();
+            if(languageId == null){
+                //Default is english:
+                languageId = 1;
+            }
+            Language lang = languageRepository.getById(languageId);
+            MemberLanguage memberLanguage = memberLanguageRepository.getByMemberIdAndLanguage(curMember.getId(), lang);
+            if(memberLanguage == null){
```

Comment:
If you use _getBy_ type of method in here it would crash instead of returning null value. Use _findBy_ clause with _findBy...().orElse(new MemberLanguage(curMember.getId(), lang));_ in the end.